packages/api/src/modules/user/user.service.ts

```diff
@@ -232,7 +232,13 @@ export class UserService {
 
         if (!user) return false;
 
-        return await userRepository.delete(user);
+        const deletePromises = [
```

Comment:
hmm ne devrait-on pas sortir le `userRepository.delete` de `Promise.all` ? Et ne lancer les deux autres qu'après celui-ci ? au cas ou il y ait une erreur

Reply:
Qu'est-ce qui n'est pas clair ?

Reply:
Qu'il faudrait mettre le `userRepository` avant les deux autres. Mais si tu penses que oui je peux faire ça. Moi j'ai pas d'avis ^^

Reply:
Je suis d'accord avec maxime, si jamais on a une erreur sur la suppression de l'utilisateur, y a de raison de supprimer le reste :)

Reply:
ok bien noté

Reply:
done